evms/parity.go

```diff
@@ -133,6 +131,12 @@ func (evm *ParityVM) Copy(out io.Writer, input io.Reader) {
 						stateRoot.StateRoot = string(root[:66])
 					}
 				}
+				// Try to unmarshall the state root
+				if err := json.Unmarshal(data, &stateRoot); err == nil {
+					if r := stateRoot.StateRoot; len(r) > 0 {
+						stateRoot.StateRoot = r
+					}
```

Comment:
??
```
if x := y; len(x) > 0 { 
    y =x 
} 
```
??

Reply:
Hahahaha sorry, I probably was a bit tired while writing this